blocks/agent-profile/agent-profile.js

```diff
@@ -79,7 +80,7 @@ const getSocialDiv = () => {
   return null;
 };
 
-export default async function decorate(block) {
+const defaultDesign = (block) => {
```

Comment:
Is there a solution where where we have a JS & CSS block renderer per design, so that once we know which design we want, we simply load those? This would reduce the amount of logic managed in this one file